vhdbuilder/packer/components.json

```diff
@@ -137,6 +137,20 @@
         "ciprod08102022"
       ]
     },
+    {
```

Comment:
why they are not multi-arch?

Reply:
@robbiezhang - We are not planning to support multi-arch for public preview.

Reply:
@robbiezhang - Updated this for kube-state-metrics. Thanks for pointing it out.